docs/platforms-release-process.md

```diff
@@ -86,30 +157,46 @@ Ensure all dependencies and subdependencies have Apache-compatible licenses.
     coho check-license -r android
 
 ## Prepare Release
-Increase the version within package.json using SemVer, and remove the `-dev` suffix.
+
+### Remove the `-dev` suffix from version
+
+This command removes `-dev` from the `version` entry in `package.json`:
 
     for l in cordova-android; do ( cd $l; v="$(grep '"version"' package.json | cut -d'"' -f4)"; if [[ $v = *-dev ]]; then v2="${v%-dev}"; echo "$l: Setting version to $v2"; sed -i '' -E 's/version":.*/version": "'$v2'",/' package.json; fi) ; done
 
-In `cordova-android`, also remember to bump the version in `framework/build.gradle`.
+Note: This command [doesn't actually work](https://issues.apache.org/jira/browse/CB-13809). You can also replace `-dev` manually of course.
 
-If the changes merit it, manually bump the major / minor/ patch version in `package.json`. View the changes via:
+Note: In `cordova-android`, also remember to handle the version in `framework/build.gradle`.
+
+### Increase version
+
+If the changes merit it, **manually** bump the major / minor/ patch version in `package.json`. 
+
+To decide if this release merits it, view the changes via:
 
     ( cd cordova-android && git log --pretty=format:'* %s' --topo-order --no-merges $(git describe --tags $(git rev-list --tags --max-count=1))..master )
 
+Note: This command [doesn't actually work](https://issues.apache.org/jira/browse/CB-13901). You can also check out the changes manually (or via the next step).
```

Comment:
once again, worked fine for me for latest android release. A pre-requisite to running this command is making sure you have the latest tags. I had to run `git fetch --tags` in cordova-android repo to grab the tags I was missing.

Reply:
Just noticed that I only ran this on Windows shell, not WSL/bash so this one might actually be just a different in the params of `git` on Windows.

Reply:
Indeed, it works in WSL. Will update the text.